doc/releases/migration-guide-3.7.rst

```diff
@@ -212,6 +212,14 @@ LED Strip
 * The property ``in-gpios`` defined in :dtcompatible:`worldsemi,ws2812-gpio` has been
   renamed to ``gpios``.
 
+* The ``chain-length`` and ``color-mapping`` properties have been added to all LED strip bindings
```

Comment:
@kartben shouldn't we have similar point in the release notes? (or at least something like "LED strips: bindings and API have been updated"

Reply:
yep

Reply:
I don't see what needs to be added, the changes are described here, they're in migration guide and not release notes because these are changes users need to be aware of, this is how it's documented to put things in the migration guide/release notes